File: fizzbuzz.py
#!/usr/bin/env python

for i in range(1,101):
    if i % 15 == 0:
        print "fizzbuzz"
    elif i % 3 == 0:
        print "fizz"
    elif i % 5 == 0:
        print "buzz"
    else:
        print i

File: fizzbuzz.rb
#!/usr/bin/env ruby

(1..100).each do |i|
    if i % 15 == 0 then
        puts "fizzbuzz"
    elsif i % 3 == 0 then
        puts "fizz"
    elsif i % 5 == 0 then
        puts "buzz"
    else
        puts i
    end
end
